oak/server/rust/oak_runtime/src/node/grpc/server.rs

```diff
@@ -131,9 +131,9 @@ impl Node for GrpcServerNode {
             // Use simple scheduler that runs all tasks on the current-thread.
             // https://docs.rs/tokio/0.2.16/tokio/runtime/index.html#basic-scheduler
             .basic_scheduler()
-            // Enables the I/O driver.
+            // Enables the I/O driver and the time driver.
             // Necessary for using net, process, signal, and I/O types on the Tokio runtime.
-            .enable_io()
+            .enable_all()
```

Comment:
I think we should call `enable_io` and `enable_time` separately. The doc says that `enable_all` will enable more components if they are added in the future: https://docs.rs/tokio/0.2.20/tokio/runtime/struct.Builder.html#method.enable_all